ui/pages/send/send-footer/send-footer.stories.js

```diff
@@ -1,30 +1,37 @@
 import React from 'react';
-import { action } from '@storybook/addon-actions';
-import { boolean } from '@storybook/addon-knobs';
 
 import SendFooter from './send-footer.component';
 
 export default {
   title: 'Pages/Send/SendFooter',
   id: __filename,
+  argTypes: {
+    clearSend: { action: 'Cancel Button Pressed' },
+    sign: { action: 'Next Button Pressed' },
```

Comment:
I think we should continue with the convention of just using the prop name for the action. That way developers have a clear understanding of which props the action is being passed to.

```suggestion
    clearSend: { action: 'clearSend' },
    sign: { action: 'sign' },
```

Reply:
ok, changed.